src/onboarding/Templates/TemplateDetails.js

```diff
@@ -32,8 +34,11 @@ function TemplateDetails({ template, visible, onUse, onClose }) {
     onUse(template.id, selectedOptionalApps)
 
     // analytics test
-    trackEvent(events.TEMPLATE_SELECTED, { name: template.name })
-  }, [onUse, template, templateOptionalApps])
+    trackEvent(events.DAOCREATION_TEMPLATE_SELECTED, {
+      name: template.name,
+      network: networkName,
```

Comment:
Wouldn´t it be better to have the network being injected inside the trackEvent function? Otherwise, any new event needs to take care of adding it

Reply:
it is not recomended to use hooks such as useWallet() inside event handlers https://reactjs.org/warnings/invalid-hook-call-warning.html

however i noticed that rudderstack has traits, that it pick up the network if the app is connected once  (but it has to be connected at least once),

if there is another way you guys can think of please let me know

Reply:
Why not storing last network on localStorage and just pull it from there? It actually will be needed for another ticket to save last network and keep app there